val-ansible-role/valkey/tasks/install.yml

```diff
@@ -0,0 +1,44 @@
+- name: Ensure dependencies are installed
+  package:
+    name: ["build-essential", "tcl", "wget"]
+    state: present
+    update_cache: yes
+
+- name: Download Valkey
```

Comment:
there is option to install using apt too
https://valkey.io/topics/installation/